test/e2e/manifests/mysql/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
```

Comment:
Note: this manifest likely won't be needed once the Hive PR, which bumps the mysql connector java version, lands and we can instead use the `mysql:8.0` imagestream that gets shipped by default in OCP.